apps/site/src/_pages/docs/0_index.mdx

```diff
@@ -2,41 +2,39 @@
+We additionally explain core concepts in the Block Protocol, including:
 
-From a developer perspective, blocks encapsulate a set of specific functionality. Building a block is no different to writing a React, Vue, or [Web Component](https://en.wikipedia.org/wiki/Web_Components) – in fact, these are all ways you might implement a block. They are reusable elements that expect certain data to be passed to them (what we refer to as a block’s ‘schema’). For example, an image block might only accept an image URL in its `src` property, not a number.
+- [Blocks](https://blockprotocol.org/docs/blocks): what they actually are, and what makes Þ blocks special
+- [Types](https://blockprotocol.org/docs/types): an overview, how to started, and how they enable next-level modeling of structured content and data
+- [Services](https://blockprotocol.org/docs/services): how external APIs and information are usable within blocks
 
-## What is a Þ block?
+If you have any questions, [please open or contribute to a discussion](https://github.com/blockprotocol/blockprotocol/discussions) on GitHub, or [chat to us on Discord](https://blockprotocol.org/discord).
 
-Any block can become a Þ block by meeting the requirements defined in the specification. These requirements define what data a block can accept and how they're allowed to interact with that data. This standardized approach makes it possible to use and embed Þ blocks across any different kind of web application, provided it supports the protocol.
+## Þ Hub & API
 
-Þ blocks can read and write data to the applications they're embedded inside without any extra setup or custom integrations (as long as both the block and embedding application adhere to the specification).
+[Blocks](https://blockprotocol.org/docs/blocks) and [types](https://blockprotocol.org/docs/types) can be published and discovered on the [Þ Hub](https://blockprotocol.org/hub), an open-source registry that anybody can use. There are two ways to access it:
 
-The Block Protocol is free to use, and any application can enable the use of Þ blocks by following the [requirements for embedders](https://blockprotocol.org/docs/using-blocks).
+1. **in-browser** by navigating to [`blockprotocol.org/hub`](https://blockprotocol.org/hub): no account is required to browse and test blocks
+1. **programmatically** from within other applications: either a user or an embedding application must create a free API key in order to query blocks from the Þ Hub via its API
 
-## What is the Þ Hub?
+The Þ Hub exists because interoperability without discoverability is of limited use.** The Þ Hub provides a single persistent, reliable, and open-source point of reference for the discovery of new Þ-compatible content. Creators only need to publish once, while embedding applications can easily discover new blocks, types and endpoints, making them instantly available to their own users.
```

Comment:
```suggestion
The Þ Hub exists because interoperability without discoverability is of limited use.** The Þ Hub provides a single persistent, reliable, and open-source point of reference for the discovery of new Þ-compatible content. Creators only need to publish once, while embedding applications can easily discover new blocks, types and services, making them instantly available to their own users.
```